actors.md

```diff
@@ -580,6 +580,38 @@ func SubmitPost(proofs []PoStProof, faults []FaultSet, recovered BitField, done
 	miner.NextDoneSet = done
 	miner.ArbitratedDeals.Clear()
 }
+
+func ValidateFaultSets(faults []FaultSet, recovered, done BitField) bool {
+	var aggregate BitField
+	for _, fs := range faults {
+		aggregate.Combine(fs.BitField)
```

Comment:
nit: `Union` might be a more intuitive name for what's going on with the bitsets here

Reply:
yeah, thats a good point